make_release/nu_release.nu

```diff
@@ -31,38 +30,26 @@ let subcrates_wave_3 = [
     nu-cli,
     nu-std,
 
-    # plugins
     nu_plugin_query,
     nu_plugin_inc,
     nu_plugin_gstat,
     nu_plugin_formats,
 ]
 
-# Recent versions of cargo verify the upload of your crate
-# So no need to `sleep` anymore.
-
+log warning "publishing the first wave of crates"
 for subcrate in $subcrates_wave_1 {
-    cd $subcrate
-    cargo publish
-    # sleep 1min
-    cd ..
+    cargo publish --manifest-path ("crates" | path join $subcrate "Cargo.toml")
```

Comment:
Have you validated that the `--manifest-path` behaves exactly the same as navigating to the directory and running there? Not that it just redefines the `Cargo.toml` location and still tries to use the local directory (my brief glance at the docs for the flag was less than satisfying in this regard)

Reply:
this is a legitimate concern :thinking: 

i agree the doc of `--manifest-path` is not super satisfactory and as it's hard to test `cargo publish`, i see two options to still have a cleaner script
- fallback to `-C <path>` which is only available on the `+nightly` toolchain :thinking: 
- use `def` commands with a single `cd <path>` in them so that the *cd back* is implicit

what do you prefer? :yum:

Reply:
@sholderbach 
i went with the *env-scoped `publish` command* in 449fea4 :crossed_fingers:

Reply:
Nice! I *think* we could directly check for the `--no-verify` condtion by checking if a `build.rs` is present which would replace the crate waves with just a single list of the DFS of the dependency tree.

Reply:
ooooh, yeah that'd be nice :thinking: 

maybe in another followup PR?